### PR TITLE
Select worktype to create using a radio button

### PR DESCRIPTION
**Pull request by jcoyne**
This also enables selecting a work type when you click the "Share my
Work" button on the homepage
Ref projecthydra-labs/hyku#580

screen shot 2017-01-20 at 1 57 49 pm

### CODE REVIEW

File: app/views/sufia/homepage/index.html.erb [hunk 1]
```diff
@@ -3,9 +3,17 @@
...
-            class: "btn btn-primary btn-lg" do %>
-        <i class="glyphicon glyphicon-upload"></i> <%= t('sufia.share_button') %>
+      <% if signed_in? %>
+        <%= link_to '#',
+          class: "btn btn-primary btn-lg",
+          data: { behavior: 'select-work', target: '#worktypes-to-create', 'create-type' => 'single' } do %>
+          <i class="glyphicon glyphicon-upload"></i> <%= t('sufia.share_button') %>
+        <% end %>
+      <% else %>
+        <%= link_to sufia.dashboard_index_path,
```
Comment (mjgiarlo) [inline app/views/sufia/homepage/index.html.erb]: Why the dashboard?

**Comment by mjgiarlo** [comment]
@ggeisler How does ⬆️ look as a starting point for taking on the CSS styling in a follow-on PR?

**Comment by ggeisler** [comment]
👍  Looks good.

File: app/views/sufia/homepage/index.html.erb (hunk 1 above)
Comment (jcoyne) [inline app/views/sufia/homepage/index.html.erb]: That's where it went before, abet more indirectly

**Comment by mjgiarlo** [comment]
Great! It's all yours now, @ggeisler. :)

**Pull request merged**